<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilterEditor</class>
 <widget class="QWidget" name="FilterEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>617</width>
    <height>419</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="m_name"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>4</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>String Matching</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="KEditListWidget" name="m_word_matches">
         <property name="toolTip">
          <string>&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;The string to find in the title of an item. Simple wildcard syntax is supported :&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;/p&gt;
&lt;table border=&quot;0&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px;&quot; align=&quot;center&quot; cellspacing=&quot;1&quot; cellpadding=&quot;2&quot;&gt;
&lt;tr&gt;
&lt;td style=&quot; vertical-align:top;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; color:#000000;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;c&lt;/span&gt;&lt;/p&gt;&lt;/td&gt;
&lt;td style=&quot; vertical-align:top;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Any character represents itself apart from those mentioned below. Thus &lt;span style=&quot; font-weight:600;&quot;&gt;c&lt;/span&gt; matches the character &lt;span style=&quot; font-style:italic;&quot;&gt;c&lt;/span&gt;.&lt;/p&gt;&lt;/td&gt;&lt;/tr&gt;
&lt;tr&gt;
&lt;td style=&quot; vertical-align:top;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; color:#000000;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;?&lt;/span&gt;&lt;/p&gt;&lt;/td&gt;
&lt;td style=&quot; vertical-align:top;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; color:#000000;&quot;&gt;Matches any single character. &lt;/p&gt;&lt;/td&gt;&lt;/tr&gt;
&lt;tr&gt;
&lt;td style=&quot; vertical-align:top;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; color:#000000;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;*&lt;/span&gt;&lt;/p&gt;&lt;/td&gt;
&lt;td style=&quot; vertical-align:top;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; color:#000000;&quot;&gt;Matches zero or more of any characters.&lt;/p&gt;&lt;/td&gt;&lt;/tr&gt;
&lt;tr&gt;
&lt;td style=&quot; vertical-align:top;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; color:#000000;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;[...]&lt;/span&gt;&lt;/p&gt;&lt;/td&gt;
&lt;td style=&quot; vertical-align:top;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; color:#000000;&quot;&gt;Sets of characters can be represented in square brackets. Within the character class, like outside, backslash has no special meaning.&lt;/p&gt;&lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; color:#000000;&quot;&gt;For example the *.torrent will match any item which ends in .torrent. &lt;/p&gt;</string>
         </property>
         <property name="buttons">
          <set>KEditListWidget::Add|KEditListWidget::Remove</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_reg_exp_syntax">
         <property name="toolTip">
          <string>By default strings will use wildcard matching. If you want to use regular expressions, this needs to be enabled.</string>
         </property>
         <property name="text">
          <string>Strings are regular expressions</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_all_words_must_match">
         <property name="text">
          <string>Item must match all strings to be included</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_match_case_sensitive">
         <property name="text">
          <string>Case sensitive matching</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>Exclusion Patterns</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="KEditListWidget" name="m_exclusion_patterns">
         <property name="toolTip">
          <string>&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;The string to find in the title of an item. Simple wildcard syntax is supported :&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;/p&gt;
&lt;table border=&quot;0&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px;&quot; align=&quot;center&quot; cellspacing=&quot;1&quot; cellpadding=&quot;2&quot;&gt;
&lt;tr&gt;
&lt;td style=&quot; vertical-align:top;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; color:#000000;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;c&lt;/span&gt;&lt;/p&gt;&lt;/td&gt;
&lt;td style=&quot; vertical-align:top;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Any character represents itself apart from those mentioned below. Thus &lt;span style=&quot; font-weight:600;&quot;&gt;c&lt;/span&gt; matches the character &lt;span style=&quot; font-style:italic;&quot;&gt;c&lt;/span&gt;.&lt;/p&gt;&lt;/td&gt;&lt;/tr&gt;
&lt;tr&gt;
&lt;td style=&quot; vertical-align:top;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; color:#000000;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;?&lt;/span&gt;&lt;/p&gt;&lt;/td&gt;
&lt;td style=&quot; vertical-align:top;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; color:#000000;&quot;&gt;Matches any single character. &lt;/p&gt;&lt;/td&gt;&lt;/tr&gt;
&lt;tr&gt;
&lt;td style=&quot; vertical-align:top;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; color:#000000;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;*&lt;/span&gt;&lt;/p&gt;&lt;/td&gt;
&lt;td style=&quot; vertical-align:top;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; color:#000000;&quot;&gt;Matches zero or more of any characters.&lt;/p&gt;&lt;/td&gt;&lt;/tr&gt;
&lt;tr&gt;
&lt;td style=&quot; vertical-align:top;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; color:#000000;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;[...]&lt;/span&gt;&lt;/p&gt;&lt;/td&gt;
&lt;td style=&quot; vertical-align:top;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; color:#000000;&quot;&gt;Sets of characters can be represented in square brackets. Within the character class, like outside, backslash has no special meaning.&lt;/p&gt;&lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; color:#000000;&quot;&gt;For example the *.torrent will match any item which ends in .torrent. &lt;/p&gt;</string>
         </property>
         <property name="buttons">
          <set>KEditListWidget::Add|KEditListWidget::Remove</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_exclusion_reg_exp">
         <property name="toolTip">
          <string>By default strings will use wildcard matching. If you want to use regular expressions, this needs to be enabled.</string>
         </property>
         <property name="text">
          <string>Strings are regular expressions</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_exclusion_all_must_match">
         <property name="text">
          <string>Item must match all strings to be excluded</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_exclusion_case_sensitive">
         <property name="text">
          <string>Case sensitive matching</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Seasons &amp;&amp; Episodes</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QCheckBox" name="m_use_se_matching">
         <property name="text">
          <string>Use season and episode matching</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Seasons: </string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="m_seasons">
           <property name="toolTip">
            <string>Specify the seasons to download, multiple seasons can be separated by a , and you can also use ranges by using a -

For example: 1, 3-6, 9

Will download seasons 1, 3, 4, 5, 6 and 9</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Episodes:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="m_episodes">
           <property name="toolTip">
            <string>Specify the episodes to download, multiple episodes can be separated by a , and you can also use ranges by using a -

For example: 1, 3-6, 9

Will download episodes 1, 3, 4, 5, 6 and 9</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="m_se_no_duplicates">
         <property name="text">
          <string>Do not download multiple items with the same season and episode</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Options</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QCheckBox" name="m_download_matches">
         <property name="text">
          <string>Download torrents of matching items</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_download_non_matches">
         <property name="text">
          <string>Download torrents of non matching items</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QCheckBox" name="m_add_to_group">
           <property name="text">
            <string>Add to group:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="m_group">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QCheckBox" name="m_use_custom_download_location">
           <property name="text">
            <string>Download to:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="KUrlRequester" name="m_custom_download_location"/>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="m_move_on_completion">
           <property name="text">
            <string>Move when completed to:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="KUrlRequester" name="m_move_on_completion_location"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="m_silently">
         <property name="text">
          <string>Open torrents silently</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>174</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Test</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QPushButton" name="m_test">
           <property name="text">
            <string>Test</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>filter on feed:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="m_feed"/>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTreeView" name="m_test_results">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KEditListWidget</class>
   <extends>QWidget</extends>
   <header>keditlistwidget.h</header>
  </customwidget>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>m_use_se_matching</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_seasons</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>104</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_use_se_matching</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_episodes</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>37</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>101</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_add_to_group</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_group</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>80</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>188</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_use_custom_download_location</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_custom_download_location</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>165</y>
    </hint>
    <hint type="destinationlabel">
     <x>336</x>
     <y>171</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_use_se_matching</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_se_no_duplicates</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>55</x>
     <y>170</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_move_on_completion</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_move_on_completion_location</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>155</x>
     <y>185</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
